sync_ci/pkg/parser/ci_log.go

```diff
@@ -58,6 +63,14 @@ func ParseCILog(job string, ID int64) (map[string][]string, error) {
 		lines = append(lines, line[0])
 		lines = lines[1:]
 	}
+
+	results := ApplyRegexpRulesToLines(job, regexRulesLines)
+	if results != nil {
+		for _, kv := range *results {
+			res[kv.Key] = append(res[kv.Key], kv.Value)
```

Comment:
For the consideration of brevity, I think duplicate values ​​need to be removed here.

Reply:
Yes, i will do it

Reply:
removed duplicates ones at the end of the process.